--- support/launch.sh	2026-04-24 00:00:00.000000000 +0000
+++ support/launch.sh.new	2026-04-24 00:00:00.000000000 +0000
@@ -179,10 +179,7 @@
 	VMARGS_FROM_LAUNCH_SH+=("-Xdock:name=${APPNAME}")
 fi

-# Set Max Heap Size if specified
-if [ "${MAXMEM}" != "" ]; then
-	VMARGS_FROM_LAUNCH_SH+=("-Xmx${MAXMEM}")
-fi
+VMARGS_FROM_LAUNCH_SH+=("-XX:InitialRAMPercentage=20" "-XX:MinRAMPercentage=20" "-XX:MaxRAMPercentage=80")

 BACKGROUND=false

